### PR TITLE
CMake: fix j2k extension support

### PR DESCRIPTION
**Pull request by kmilos**

### CODE REVIEW

File: src/CMakeLists.txt [hunk 1]
```diff
@@ -580,7 +581,6 @@ endif(LensFun_FOUND)
 
 if(OpenEXR_FOUND)
   add_definitions("-DHAVE_OPENEXR")
-  set(DT_SUPPORTED_EXTENSIONS ${DT_SUPPORTED_EXTENSIONS} j2c j2k jp2 jpc CACHE INTERNAL "")
```
Comment (TurboGit) [inline src/CMakeLists.txt:583]: don't we want here to add he `exr` extension?

Comment (kmilos) [reply src/CMakeLists.txt:583]: It's already always on in the default list, but maybe indeed it shouldn't be...

Comment (TurboGit) [reply src/CMakeLists.txt:583]: > It's already always on in the default list, but maybe indeed it shouldn't be...

Probably and I'll do so. Looking at the history it seems that adding it by default was an oversight.

**Review by TurboGit — APPROVED** [review]
Thanks!

**Pull request merged**